src/providers/zoom.js

```diff
@@ -0,0 +1,21 @@
+export default function Zoom(options) {
+  return {
+    id: "zoom",
+    name: "zoom",
```

Comment:
Looks like they are branding with a capital Z, let us respect that here as well.
```suggestion
    name: "Zoom",
```